src/main/java/me/earth/earthhack/impl/modules/misc/tracker/ListenerTick.java

```diff
@@ -20,99 +20,96 @@ public ListenerTick(Tracker module)
     @Override
     public void invoke(TickEvent event)
     {
-        if (event.isSafe())
+        if (event.isSafe() && module.isEnabled())
```

Comment:
Can you remove `module.isEnabled()` check? The method wont be invoked if the module is disabled.